lib/agent/https_proxy_ocsp_agent.js

```diff
@@ -98,7 +98,25 @@ function connect(req, opts, fn)
 {
   var proxy = this.proxy;
   var agent = this;
-  Logger.getInstance().debug("Using proxy=%s for host %s", proxy.host, opts.host);
+
+  // try and detect if there's proxy envvar set, currently it's not logged anywhere
```

Comment:
do we support env variables for proxy in NodeJs driver?

Reply:
we do not, to my knowledge. 
that does not keep users from trying to still use it (most of the times inadvertently, e.g. it's already set in the env for other application. or trying to use it since our other drivers support it), then get in all sorts of trouble, ending up with Snowflake Support who now doesn't have a really easy way of confirming this

Reply:
should we do anything more than only reading it? eomeone who reads logs will see that maybe there is some difference between those options and will it be enough?

Here we are using agent that is aware of proxy so we can see only differences between driver and env configuration - maybe checking proxy env variables should also be done when we don't have proxy configuration on connection?

Reply:
thank you - i'll work on this per our discussion

Reply:
re: the env variables: even though it is not documented yet, my tests show that we do support it. I mean the connection breaks as soon as I point HTTP_PROXY / HTTPS_PROXY to a non-existing proxy. It can be confirmed it _does_ go through the proxy as soon as I set up a proxy and set HTTPS_PROXY to point the driver towards it.

Even without proxyHost/proxyPort.

Reply:
PROXY envvar now also should be detected outside of the agent